addon/templates/components/md-nav.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="nav-wrapper">
+  <div class="container">
```

Comment:
Depends if you want your navbar contents aligned with the body content. No other styles should come along. Let me know if you find issues and I'll go fix them upstream in Materialize's SASS